### PR TITLE
Added scraping of prometheus endpoint to Docker stacks

### PR DESCRIPTION
**Pull request by fkaleo**

### CODE REVIEW

File: docker/docker-cluster/prometheus/prometheus.yml [hunk 1]
```diff
@@ -26,3 +26,9 @@ scrape_configs:
     scrape_interval: 15s
     static_configs:
       - targets: ['localhost:9090']
+
+  - job_name: metrictank
+    metrics_path: /prometheus/metrics
+    scrape_interval: 10s
+    static_configs:
+      - targets: ['metrictank:6060']
```
Comment (Dieterbe) [inline docker/docker-cluster/prometheus/prometheus.yml]: for the clustered ones, you'll probably want to scrape every metrictank instance 
in that case, i'm not sure if you can differentiate the different metrics (do they all get a unique host tag? or does it require separate scrape jobs? you'll need to try or ask a prometheus person)

Comment (fkaleo) [reply docker/docker-cluster/prometheus/prometheus.yml]: Sorry, I was careless there. I'll figure that out.

Comment (fkaleo) [reply docker/docker-cluster/prometheus/prometheus.yml]: Prometheus adds a label per host :)

image

Comment (fkaleo) [reply docker/docker-cluster/prometheus/prometheus.yml]: Fixed in next commit

File: docker/docker-cluster-query/prometheus/prometheus.yml [hunk 2]
```diff
@@ -26,3 +26,9 @@ scrape_configs:
     scrape_interval: 15s
     static_configs:
       - targets: ['localhost:9090']
+
+  - job_name: metrictank
+    metrics_path: /prometheus/metrics
+    scrape_interval: 10s
+    static_configs:
+      - targets: ['metrictank0:6060','metrictank1:6060','metrictank2:6060','metrictank3:6060']
```
Comment (Dieterbe) [inline docker/docker-cluster-query/prometheus/prometheus.yml]:
```suggestion
      - targets: ['metrictank0:6060','metrictank1:6060','metrictank2:6060','metrictank3:6060','metrictank-q0:6060']
```

**Pull request merged**